backend/public-graph/graph/resolver.go

```diff
@@ -784,8 +743,8 @@ func (r *Resolver) HandleErrorAndGroup(ctx context.Context, errorObj *model.Erro
 	if errorObj == nil {
 		return nil, e.New("error object was nil")
 	}
-	if errorObj.Event == "" || errorObj.Event == "<nil>" {
```

Comment:
Tracked this back to #1684. The original slack [message](https://highlightcorp.slack.com/archives/C01R04YNCEA/p1635803106004500) has no context as to why we are actually doing it. Would rather remove defensive guards unless we have a reason because this code prevents someone from seeing an error message of `"<nil>"` which (while silly) is totally a legitimate value.